epochX/cudacpp/gg_tt.mad/SubProcesses/runTest.cc

```diff
@@ -222,9 +230,20 @@ struct CUDATest : public CUDA_CPU_TestBase
     constexpr fptype fixedG = 1.2177157847767195; // fixed G for aS=0.118 (hardcoded for now in check_sa.cc, fcheck_sa.f, runTest.cc)
     for( unsigned int i = 0; i < nevt; ++i ) hstGs[i] = fixedG;
     copyDeviceFromHost( devGs, hstGs ); // BUG FIX #566
+<<<<<<< HEAD
```

Comment:
This PR was merged including merge conflicts... See #901

Reply:
This is related to the re-generation of the code for our standard example processes, IIRC we touched on this during the dev meeting yesterday.